src/components/Fade.tsx

```diff
@@ -16,8 +16,8 @@ const fade = (bottom = true) =>
       bottom ? 'to-primary-dark' : 'to-transparent'
     ),
     position('sticky'),
-    inset(bottom ? '-bottom-1' : '-top-1', 'left-0', 'right-0'),
-    height('h-4'),
+    inset(bottom ? '-bottom-2' : '-top-1', 'left-0', 'right-0'),
+    height(bottom ? 'h-6' : 'h-4'),
```

Comment:
Bottom fade is now bigger, this fixes this thin light line, that appeared between shadow and content